.github/workflows/itk_dict.txt

```diff
@@ -0,0 +1,997 @@
+Wachowiak
+Wanlin
+Wi
+Wtautological
+Xu
+Y'CbCr
+Yager's
+Yixun
+Yhis
+Yuanting
+Zeiss
+Zosso
+Zurada
+aPrioris
+acq
+acqp
+advective
+aic
+algo
+alloc
+analyse
+anisotropies
+annuli
+anticausal
+antiraster
+ap
+apath
+archetypical
+argidx
+ascii
+asymptote
+asymptotes
+atfork
+attenuator
+autocrop
+backend
+backends
+bayes
+bb
+bestdistortion
+bezier
+binarization
+binsize
+bistoury
+bg
+blockings
+brainweb
+branchness
+bresenham
+bsplice
+btw
+buf
+bugfix
+bw
+byu
+bzip
+calc
+calloc
+callosum
+centerline
+cdf
+cdfs
+cfl
+cgi
+cha
+chebycoefs
+chebyshev
+chokkan
+chroma
+ci
+classvtk
+cline
+cme
+cms
+codec
+coeff
+coeffP
+composedness
+complainty
+computable
+connectedness
+convolves
+confocal
+cooccurrence
+costless
+cp
+cr
+cranio
+creativecommons
+cryo
+cryosection
+cryosectional
+ctwd
+cumulate
+cumulated
+curvilinear
+cv
+cvCvtColor
+cvals
+cx
+cy
+d'Attente
+dDwWmMyY
+da
+dat
+dcmimage
+dcmtk
+datablock
+dataig
+dataobject
+datapipeline
+dataraces
+dbg
+ddu
+deallocated
+decompositions
+decrementing
+decubitus
+defacto
+defe
+defem
+deformably
+deformer
+degr
+demark
+denoiser
+denormal
+deriv
+descr
+detections
+dgr
+dic
+dicoopxt
+dicts
+dilations
+dima
+dimb
+dimensionally
+dimensioned
+dirn
+dirns
+dl
+docstring
+dodecahedron
+dof
+dom
+downlist
+downlists
+doublereal
+drand
+dseq
+dtnrm
+dtwd
+duplications
+dwi
+dxx
+dxy
+dxz
+dyy
+dyz
+dzz
+e'e
+echall
+edgecell
+eg
+eigenVecs
+eigensolve
+eigenvec
+eix
+eiy
+eiz
+el
+elt
+elmts
+emph
+endlink
+ent
+entropies
+eps
+epsmch
+epsmcj
+eqn
+equalities
+equispaced
+eqi
+eqrt
+equs
+erf
+erfc
+etd
+evaluable
+excitations
+explicits
+extensionkey
+extradiagonal
+extrapolators
+extremum
+ey
+ez
+factorable
+fah
+failsafe
+fastmarching
+favourite
+fcv
+fdinbuf
+fdistream
+fdostream
+fdstream
+femobject
+fh
+fftinput
+fftoutput
+fgm
+fi
+fia
+fid
+fifo
+floatingfigure
+fn
+fnorm
+formatter
+formatters
+fortran
+fourier
+framebox
+framerate
+freesurfer
+frobenius
+fsa
+fsb
+fstream
+ftol
+functors
+fw
+fx
+fy
+ga
+galist
+gamold
+gauss
+gaussians
+gb
+gc
+gdcm
+gdef
+geometrydata
+gerrit
+gifti
+gifticlib
+gii
+glehmann
+gnuplot
+gp
+gpu
+greyval
+guid
+gtol
+gzip
+halfwidth
+haralick
+hardcoding
+hdfgroup
+hexa
+hexahedral
+hh
+hhs
+highrange
+hipaa
+hor
+hough
+hval
+hvector
+ia
+ibmth
+icall
+icnt
+icosahedron
+idtype
+idx'th
+iems
+ier
+ierr
+ierr'th
+ifac
+ifstream
+ifti
+ijk
+imagedimension
+imit
+immunofluorescent
+imsl
+imthd
+imthdd
+includegraphics
+incx
+incy
+indct
+indir
+inhomogeneous
+initialise
+initialised
+initialisation
+initializations
+initnorm
+inpMax
+inpMin
+inputreader
+intensityrescaler
+interactor
+interdependencies
+interslice
+intra
+invariance
+invariants
+ipl
+isValidJacobianCalcLocat
+isocontouring
+isol
+isk
+iter
+iterator's
+itkGEAdwImageIO
+itkFEMLoadGrav
+itkcaption
+itmax
+itpack
+ip
+iparm
+ipstr
+isw
+itr
+ival
+iwksp
+iwork
+iwskp
+ja
+jacobi
+jacobians
+jcd
+jcg
+jh
+jj
+jsi
+jvm
+k'th
+kHd
+kitware
+kwargs
+kwstyle
+kmeans
+kth
+labeler
+labeller
```

Comment:
Should this be removed? There is `labeler` on the previous line.

Reply:
It seems that 'labelled' is acceptable British spelling, and so I guess 'labeller' ought to be ok too.  The code has a lot of uses of 'labelled' in it, even in method and variable names.